lib/src/lib.nr

```diff
@@ -89,10 +89,7 @@ impl<PROOF_LEN, MAX_VALUE_LEN> TrieProof<32, PROOF_LEN, MAX_VALUE_LEN>
             let k = if (in_range) {i} else {0}; // Restrict index to range {0, ..., depth - 2}
 
             // Populate node array
-	        for j in 0..MAX_TRIE_NODE_LENGTH
-	        {
-		        node[j] = path[j + k*MAX_TRIE_NODE_LENGTH];
-	        }
+            memcpy(&mut node, path, k*MAX_TRIE_NODE_LENGTH);
```

Comment:
Let's add spaces around operators. This is something 99% of people agree on and I'm pretty sure it's an oversight on his side as he does it in other places